styles.css

```diff
@@ -68,14 +68,29 @@ h3 {
     border-radius: 10px;
     padding-top: 11px;
     position: absolute;
-    top: 80px;
     left: -110px;
 }
 
-.sideBox div {
+.sideBox1 div {
     font-size: 40px;
 }
 
+.sideBox2 div {
+    font-size: 25px;
+}
+
+.sideBox1 {
+    top: 80px;
+}
+
+.sideBox2 {
+    top: 190px;
+}
+
+.sideBox2 h3 {
+    margin-bottom: 11px;
+}
```

Comment:
To make the css cleaner, you could add this to .sidebox2 div as a top margin, instead of overwriting the h3 margin